spaces/domains.json

```
@@ -262,5 +262,6 @@
   "vote.frenchie.so": "frenchienetwork.eth",
   "gov.50cent.network": "50centnetwork.eth",
   "vote.kogecoin.io": "kogecoin.eth",
-  "vote.tea-swap.finance": "tea-swap.eth"
+  "vote.tea-swap.finance": "tea-swap.eth",
+  "voting.chimpyswap.com"
```

Your domain field value should be similar to your space directory `"my.custom.domain": "my-space.eth"`.
```suggestion
  "voting.chimpyswap.com": "chimpyswap.eth"
```